.github/workflows/lighthouse.yml

```
@@ -89,13 +89,15 @@ jobs:
         with:
           path: ./build/*
           key: ${{ github.sha }}
+      - name: Install wp-env
+        shell: bash
+        run: |
+              npm i -g --no-package-lock --no-save @wordpress/env@8.6.1
       - name: Run wp-env
         if: matrix.wpCoreVersion != 'master'
-        uses: ./.github/workflows/run-wp-env
-        with:
-          TEMPLATES: './tests/lighthouse/templates'
-          PHP_VERSION: ${{ matrix.phpVersion }}
-          WP_CORE_VERSION: ${{ matrix.wpCoreVersion }}
+        run: |
+          npx wp-env start
+          npx wp-env run cli bash elementor-config/setup.sh
```

Lets use centralized scripts to avoid differences between CI and local development
```suggestion
          npm run start-local-server
          npm run test:setup:playwright
```

---

done